src/main/java/teammates/common/util/Const.java

```diff
@@ -613,11 +613,26 @@ private Const() {
         public static final String INSTRUCTOR_PERMISSION_ROLE_TUTOR = "Tutor";
         public static final String INSTRUCTOR_PERMISSION_ROLE_CUSTOM = "Custom";
     }
+    /**
+     *  Gender that can be used.
+     *  <li>{@link #MALE}</li>
+     *  <li>{@link #FEMALE}</li>
+     *  <li>{@link #OTHER}</li>
+     */
 
-    public static class GenderTypes {
-        public static final String MALE = "male";
-        public static final String FEMALE = "female";
-        public static final String OTHER = "other";
+    public enum Gender {
+     /**
+      *MALE.
+      */
+     MALE,
+     /**
+      *FEMALE.
+      */
+     FEMALE,
+     /**
+      * OTHER.
+      */
+     OTHER
```

Comment:
The comments do not value-add; remove them.
Also, the values can all be in one line.
```java
public enum Gender {
    MALE, FEMALE, OTHER
}
```